treeherder/webapp/api/performance_data.py

```diff
@@ -122,6 +122,9 @@ def list(self, request, project):
 
             ret[signature_hash].update(json.loads(extra_properties))
 
+            if extra_options:
+                ret[signature_hash]['extra_options'] = extra_options
```

Comment:
Ok so just to confirm. We are storing the new extra_properties in the db as a charField (i.e. 'e10s shell'), however you want the api to not return a char, but to return a list (i.e. ['e10s', 'shell']) instead?